app/src/main/res/layout/activity_call.xml

```diff
@@ -55,6 +55,21 @@
         app:layout_constraintTop_toBottomOf="@+id/caller_number"
         tools:text="Is Calling" />
 
+    <TextView
+        android:id="@+id/hold_status_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/small_margin"
+        android:alpha="0.8"
+        android:text="@string/call_on_hold"
+        android:textColor="@color/md_red"
+        android:textSize="@dimen/call_status_text_size"
+        android:visibility="gone"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/call_status_label"
+        tools:visibility="visible" />
```

Comment:
I have to use another icon then. I cannot set image padding because it makes the phone icon too small. 
Setting the padding programmatically may solve this, but also will clog the code.

Reply:
you can create a drawable file with insets as margins there, just check https://developer.android.com/guide/topics/resources/drawable-resource#Inset

Reply:
Good idea. 
I added `ic_pause_inset` xml file, not sure about naming.